docs/lux-backend-repo-inventory.md

```diff
@@ -24,7 +24,7 @@ What follows are brief introductory descriptions of various sub-directories with
 |                                |                    |                                     |                                                       | [/lib](/src/main/ml-modules/root/lib)                                 | The heart of LUX's backend implementation, where developers get to spend most of their time when they're lucky :)                                                                           |                                                                                                                                                                                               |                                                                                                                                                                                                                           |
-|                                |                    |                                     |                                                       | [/runDuringDeployment](/src/main/ml-modules/root/runDuringDeployment) | Includes scripts to deploy then execute during deployment, directly supporting generators for the data constants, remaining search terms, related lists, and advanced search configuration. | [LUX Gradle Tasks](/docs/lux-backend-build-tool-and-tasks.md#lux-gradle-tasks), [Data Constants](/docs/lux-backend-data-constants.md)                                                         |                                                                                                                                                                                                                           |
+|                                |                    |                                     |                                                       | [/runDuringDeployment](/src/main/ml-modules/root/runDuringDeployment) | Includes scripts to deploy then execute during deployment, directly supporting generators for the remaining search terms, related lists, and advanced search configuration. | [LUX Gradle Tasks](/docs/lux-backend-build-tool-and-tasks.md#lux-gradle-tasks)                                                         |                                                                                                                                                                                                                           |

```

Comment:
My commit will include editing the source: lux-markdown-tables.xlsx.  There was an issue with the columns too.